proposals/infra/INF-0002-root-signature-in-clang.md

```diff
@@ -304,15 +304,147 @@ elimation has completed.
 
 ### Validations in Sema
 
-TODO
+#### All the values should be legal.
+
+  ShaderVisibility
+
+      'SHADER_VISIBILITY_ALL'
+      'SHADER_VISIBILITY_VERTEX'
+      'SHADER_VISIBILITY_HULL'
+      'SHADER_VISIBILITY_DOMAIN'
+      'SHADER_VISIBILITY_GEOMETRY'
+      'SHADER_VISIBILITY_PIXEL'
+      'SHADER_VISIBILITY_AMPLIFICATION'
+      'SHADER_VISIBILITY_MESH'
```

Comment:
There's no way to specify an invalid shader visibility, is there?  I think some of these should fall under "it must be syntactically correct".  You found a grammar somewhere for the root signature DSL?  Maybe we can include that in this section, or link to it?

Reply:
Added grammar.